api/lib/opentelemetry/version.rb

```diff
@@ -6,5 +6,5 @@
 
 module OpenTelemetry
   ## Current OpenTelemetry version
-  VERSION = '0.6.0'
+  VERSION = '0.6.1'
```

Comment:
API should have been a “major” version bump due to https://github.com/open-telemetry/opentelemetry-ruby/pull/411. I think I’m going to redo this with a bump to the same version for all gems.

Reply:
SGTM